packages/office-ui-fabric-react/src/components/Facepile/Facepile.base.tsx

```diff
@@ -190,14 +190,19 @@ export class FacepileBase extends BaseComponent<IFacepileProps, {}> {
     }
 
     const personaNames: string = personasOverflow.map((p: IFacepilePersona) => p.personaName).join(', ');
+    const overflowButtonTitle: string | undefined = overflowButtonProps && overflowButtonProps.title;
     const numPersonasNotPictured: number = Math.max(personasOverflow.length, 0);
 
     const { _classNames } = this;
 
     return (
-      <FacepileButton {...overflowButtonProps} ariaDescription={personaNames} className={_classNames.descriptiveOverflowButton}>
+      <FacepileButton
+        {...overflowButtonProps}
+        ariaDescription={overflowButtonTitle ? undefined : personaNames}
```

Comment:
I need to read-up on the ARIA spec here. I understand that providing an `aria-description` AND `title` _may_ be redundant, but I'd like to confirm that's the case. If it isn't flagged by ARIA validators perhaps we should not evaluate to `undefined` if `overflowButtonTitle` is provided.

Reply:
I looked into this and it turns out we use `ariaDescription` in a [number of places](https://github.com/OfficeDev/office-ui-fabric-react/search?l=TypeScript&q=ariaDescription) which _may not be_ correct since I can't find that attribute in the [ARIA spec](https://www.w3.org/TR/wai-aria-1.1/). I will open a separate issue to address that across the project today.

Coming back to this PR, I'm unsure of what to advise here. What you have is likely fine and isolated to the usage of `overflowButtonTitle`. When we refactor to use something more appropriate like `aria-label` we can test for redundancy. I _don't think_ redundancy will be an issue with `aria-description` if it is not a supported attribute.

I'm happy this PR raised awareness for our usage of `aria-description` 😄, thank you.